surf/src/diff.rs

```diff
@@ -310,7 +323,7 @@ impl Diff {
 
                                 old_entry_opt = old_iter.next();
                                 new_entry_opt = new_iter.next();
-                            },
+                            }
```

Comment:
Can we avoid these formatting changes? :)

Reply:
Ah, they snuck in this time. Is there any particular reason we don't want them in? Cause I have to cherry pick each time I make a new change, or do you have a custom config for your fmt?

Reply:
Ya there's a custom config here https://github.com/radicle-dev/radicle-surf/blob/master/.rustfmt.toml :)

I think you'd need to run with `cargo +nightly`